erica_app/tests/request_processing/test_eric_mapper.py

```diff
@@ -0,0 +1,725 @@
+        standard_est_input_data.person_b_has_merkzeichen_ag = True
+        standard_est_input_data.person_b_requests_pauschbetrag = True
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_b_pauschbetrag_has_merkzeichen_bl_tbl_h_pflegegrad is None
+
+
+class TestEstDataDisabilityDegree:
+
+    def test_if_person_a_does_not_request_pauschbetrag_and_has_disability_degree_then_do_not_set_disability_degree(self, standard_est_input_data):
+        standard_est_input_data.person_a_requests_pauschbetrag = False
+        standard_est_input_data.person_a_disability_degree = 80
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_a_pauschbetrag_disability_degree is None
+
+    def test_if_person_b_does_not_request_pauschbetrag_and_has_disability_degree_then_do_not_set_disability_degree(self, standard_est_input_data):
+        standard_est_input_data.person_b_requests_pauschbetrag = False
+        standard_est_input_data.person_b_disability_degree = 80
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_b_pauschbetrag_disability_degree is None
+
+    def test_if_person_a_disability_degree_below_20_then_do_not_set_disability_degree(self, standard_est_input_data):
+        standard_est_input_data.person_a_disability_degree = 15
+        standard_est_input_data.person_a_requests_pauschbetrag = True
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_a_pauschbetrag_disability_degree is None
+
+    def test_if_person_b_disability_degree_below_20_then_do_not_set_disability_degree(self, standard_est_input_data):
+        standard_est_input_data.person_b_disability_degree = 15
+        standard_est_input_data.person_a_requests_pauschbetrag = True
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_b_pauschbetrag_disability_degree is None
+
+    def test_if_person_a_disability_degree_20_then_set_disability_degree(self, standard_est_input_data):
+        standard_est_input_data.person_a_disability_degree = 20
+        standard_est_input_data.person_a_requests_pauschbetrag = True
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_a_pauschbetrag_disability_degree == 20
+
+    def test_if_person_b_disability_degree_20_then_set_disability_degree(self, standard_est_input_data):
+        standard_est_input_data.person_b_disability_degree = 20
+        standard_est_input_data.person_a_requests_pauschbetrag = True
+
+        resulting_input_data = EstEricMapping.parse_obj(standard_est_input_data)
+
+        assert resulting_input_data.person_b_pauschbetrag_disability_degree == 20
```

Comment:
There are some tests with the same code, but a different test name... is this on purpose?

Reply:
Maybe we could talk about this on Monday as I do not see which tests you mean